pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -466,3 +477,37 @@ func (c *Controller) updateGKENetworkParamSetStatus(ctx context.Context, params
 	}
 	return nil
 }
+
+// syncDefaultPodRanges updates the "default" params with new Pod range if needed
```

Comment:
add a more detailed explanation, what is the "if needed" reason?

Reply:
Done.